catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -586,6 +537,20 @@ private void buildFeatureFilters(List<FeatureTypeType> featureTypes, List<String
         "Wfs Source {}: Number of validated Features = {}", getId(), featureTypeFilters.size());
   }
 
+  private MetacardMapper getMetacardMapper(final QName featureTypeName) {
+    return metacardMappers
+        .stream()
+        .filter(mapper -> mapper.getFeatureType().equals(featureTypeName.toString()))
+        .findAny()
+        .orElseGet(
+            () -> {
```

Comment:
I would consider pulling this in a separate method called `createDefaultMetacardMapper`, which includes the debug statement, to improve readability,

Reply:
I've thought about this and I think it makes more sense to keep it here. The log message will lose its context if it's moved to a separate method (i.e. it would no longer be clear what "Could not find a MetacardMapper" means if, in that method, we are not attempting to find one).